FrontEnd/src/assets/scss/utils/_mixins.scss

```diff
@@ -93,3 +93,22 @@
 @mixin circular-shape {
   border-radius: 50%;
 }
+
+@mixin text-overflow-ellipsis {
+  text-overflow: ellipsis;
+  // the below styles are necessary for text-overflow: ellipsis to work
+  display: block;
+  white-space: nowrap;
+  overflow: hidden;
+}
+
+@mixin svg-fill($color, $width: 1.5rem, $height: 1.5rem) {
+  svg {
+    width: $width;
+    height: $height;
+    path {
+      fill: $color;
+      stroke: none;
```

Comment:
honestly might be good to have a parameter for this as well and have the default be none